### PR TITLE
Port to 20.04

### PR DESCRIPTION
**Pull request by rakshith-ravi**
I couldn't find a replacement for these packages:

```
libcloog-isl4
libelektra-dev
libiculx60
libmariadbclient18
libntdb1
libpolkit-backend-1-0
libsane-extras
libsane-extras-common
libv8-3.14.5
libv8-dev
vdpau-va-driver
```

Any idea on what to do about these packages? The docker image builds successfully without them though.

@jyn514 @nellshamrell

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
Locally, running 20.04, I see libmariadbclient-dev -- can you check that again? I can confirm that cursory searches do not yield results for the others.

**Comment by rakshith-ravi** [comment]
Oh yeah, I forgot to mention that. The `libmariadbclient-dev` package exists, but seems to be clashing with `libmysqlclient-dev`. It wouldn't let me install them together.

**Comment by rakshith-ravi** [comment]
Okay hold on, I think I may have confused the packages. `libmariadbclient-dev` works, but `libmariadbclient18` doesn't seem to have an installation candidate. In fact, my PR __has__ the `libmariadbclient-dev` and `libmariadbclient-dev-compat` package in it.

Sorry about the confusion

**Comment by jyn514** [comment]
> libmariadbclient18

This is left over from the initial commit, so I think as long as we have the development headers it's fine not to keep it.

> libcloog-isl4
> libntdb1

As mentioned in https://github.com/rust-lang/crates-build-env/pull/42#issuecomment-619637839 I think it's fine to get rid of these.

> libelektra-dev

This has not be packaged upstream, see https://github.com/ElektraInitiative/libelektra/issues/3412. I think it's fine not to keep it in the meantime, we can add it back later if/when it gets packaged.

> libiculx60
> libpolkit-backend-1-0
> libv8-3.14.5
> libv8dev
> vdpau-va-driver

These is also from the initial commit, I'm ok with not keeping them. There doesn't seem to be a replacement package in 20.04.

> libsane-extras
> libsane-extras-common

These appear to have been renamed to `libsane` and `libsane-common`, try adding those.

**Comment by rakshith-ravi** [comment]
Done. `libsane` and `libsane-common` is added. Not sure why the travis build was failing though. Something about a task not found.

**Comment by jyn514** [comment]
The error is
> Lint error: packages.txt is not sorted
You can fix it with `sort packages.txt > tmp && mv tmp packages.txt`.

File: packages.txt [hunk 1]
```diff
@@ -560,44 +557,45 @@ libnspr4
...
 libnss-mdns
 libnss-systemd
-libntdb1
 libnuma1
+libobjc-10-dev
 libobjc4
-libobjc-6-dev
 libobjc-7-dev
+libobjc-8-dev
+libobjc-9-dev
```
Comment (jyn514) [inline packages.txt:565]: Ditto that we only need 1, maybe `libobjc-10-dev` since it's the latest.

File: packages.txt [hunk 2]
```diff
@@ -770,21 +766,23 @@ libsqlite3-dev
...
+libssl1.1
 libssl1.1
 libssl-dev
 libssl-doc
+libstdc++-10-dev
 libstdc++6
-libstdc++-6-dev
 libstdc++-7-dev
+libstdc++-8-dev
+libstdc++-9-dev
```
Comment (jyn514) [inline packages.txt:777]: Let's only have `libstdc++-9-dev`.

File: packages.txt [hunk 3]
```diff
@@ -1011,11 +1007,11 @@ python3-distutils
 python3-gi
 python3-lib2to3
 python3-minimal
+python3-minimal
```
Comment (jyn514) [inline packages.txt:1010]: Please remove this duplicate.
```suggestion
```

Actually, I should probably add this to CI.

File: packages.txt (hunk 2 above)
Comment (rakshith-ravi) [reply packages.txt:777]: There's `libstdc++-10-dev` above that. I'll retain just that, since it's the latest?

**Comment by rakshith-ravi** [comment]
Updated

**Comment by jyn514** [comment]
Thanks so much for working on this!

**Pull request merged**